ui-tests/tests/jupytereverywhere.spec.ts

```diff
@@ -288,10 +288,13 @@ test('Should remove View Only banner when the Create Copy button is clicked', as
 });
 
 test.describe('Landing page', () => {
+  test.describe.configure({ retries: 2 });
   test('Should render the landing page as expected', async ({ page }) => {
     await page.goto('index.html');
     await page.waitForSelector('.je-hero');
 
+    await page.waitForTimeout(10000);
```

Comment:
Do we need 10 seconds? Would 2 seconds work?

```suggestion
    await page.waitForTimeout(2000);
```